test/case/infix_services/services_basic.py

```diff
@@ -11,15 +11,13 @@
 
 import time
 import infamy
-from infamy.ssdp import SsdpClient
```

Comment:
Should not the infamy ssdp-module be removed as well since there are no support for it in infix left.

Reply:
The `infamy/ssdp.py` is removed at the end of the diff, I guess you missed it?